crates/matrix-sdk/src/widget/settings/element_call.rs

```diff
@@ -205,6 +207,31 @@ impl WidgetSettings {
     }
 }
 
+/// The Capabilities required to run a element call widget. This is intended to
+/// be used in combination with: `acquire_capabilities` of the
+/// `CapabilitiesProvider`.
+pub fn get_element_call_required_permissions() -> Capabilities {
```

Comment:
Let's inline this into the bindings. This really doesn't belong in any of those two places. It belongs into the clients or a separate Element crate, but let's at least put it into the bindings for now.